Tests/SystemTests/MachPortTests.swift

```diff
@@ -19,126 +19,24 @@ import System
 #endif
 
 final class MachPortTests: XCTestCase {
-    func refCountForMachPortName(name:mach_port_name_t, kind:mach_port_right_t) -> mach_port_urefs_t {
-        var refCount:mach_port_urefs_t = 0
-        withUnsafeMutablePointer(to: &refCount) { refCount in
-            let kr = mach_port_get_refs(mach_task_self_, name, kind, refCount)
-            assert(kr == KERN_SUCCESS)
-        }
-        return refCount
-    }
-
-    func scopedReceiveRight(name:mach_port_name_t) -> mach_port_urefs_t {
-        _ = Mach.Port<Mach.ReceiveRight>(name:name) // this should automatically deallocate when going out of scope
-        return refCountForMachPortName(name:name, kind:MACH_PORT_RIGHT_RECEIVE)
-    }
-
-    func testRecieveRightDeallocation() throws {
-        var name:mach_port_name_t = 0 // Never read
-        withUnsafeMutablePointer(to:&name) { name in
-            let kr = mach_port_allocate(mach_task_self_, MACH_PORT_RIGHT_RECEIVE, name)
-            assert(kr == KERN_SUCCESS)
-        }
-
-        XCTAssert(name != 0xFFFFFFFF)
-
-        let one = scopedReceiveRight(name:name)
-        let zero = refCountForMachPortName(name:name, kind: MACH_PORT_RIGHT_RECEIVE)
-
-        XCTAssert(one == 1);
-        XCTAssert(zero == 0);
-    }
-
-    func consumeSendRightAutomatically(name:mach_port_name_t) -> mach_port_urefs_t {
-        let send = Mach.Port<Mach.SendRight>(name:name) // this should automatically deallocate when going out of scope
-        return send.withBorrowedName { name in
-            // Get the ref count before automatic deallocation happens
-            return refCountForMachPortName(name:name, kind:MACH_PORT_RIGHT_SEND)
-        }
-    }
-
-    func testSendRightDeallocation() throws {
-        let recv = Mach.Port<Mach.ReceiveRight>()
-        recv.withBorrowedName { name in
-            let kr = mach_port_insert_right(mach_task_self_, name, name, mach_msg_type_name_t(MACH_MSG_TYPE_MAKE_SEND))
-            XCTAssert(kr == KERN_SUCCESS)
-            let one = consumeSendRightAutomatically(name:name)
-            XCTAssert(one == 1);
-            let zero = refCountForMachPortName(name:name, kind:MACH_PORT_RIGHT_SEND)
-            XCTAssert(zero == 0);
-        }
-    }
-
-    func testSendRightRelinquishment() throws {
```

Comment:
Why do we need to get rid of all of these tests?

Reply:
We don't; this is in a commit marked `do not merge`. The underlying functionality hits an error, though.